packages/siri-sx-generator/package.json

```diff
@@ -7,7 +7,8 @@
         "test:unit": "vitest",
         "test:updateSnapshots": "vitest -u",
         "typecheck": "tsc -noEmit",
-        "dynamo-publish-json": "node --experimental-specifier-resolution=node --experimental-modules --no-warnings --loader ts-node/esm  test/siriSXUploadToDynamo.ts"
+        "dynamo-publish-json": "node --experimental-specifier-resolution=node --experimental-modules --no-warnings --loader ts-node/esm  test/siriSXUploadToDynamo.ts",
+        "dynamo-publish-site-disruptions": "node --experimental-specifier-resolution=node --experimental-modules --no-warnings --loader ts-node/esm test/uploadTestDisruptions.ts"
```

Comment:
When I change something e.g. description and then leave it empty press save and then correct it the next pages are not pre-populated